modules/mailing/Mailing.php

```diff
@@ -1294,7 +1294,7 @@ public function processIncomingEmailAccount(IncomingEmailAccount $account)
                                     $issue->setTitle(mb_decode_mimeheader($email->subject));
                                     $issue->setDescription($data);
                                     $issue->setPostedBy($user);
-                                    $issue->setIssuetype($account->getIssuetype());
+                                    $issue->setIssuetype($account->getIssuetype()->getID());
```

Comment:
Was this not working before? The underlying libraries would handle turning objects into their respective ID's when saving, and the the `setIssuetype()` method should handle receiving actual issue types.

Reply:
No, it wasn't. B2DB was throwing an error, where it could not convert from object (or array, can't remember) to a number or string.